processors/hra/extractors.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .. import base
+
+
+# Module API
+
+def extract_source(record):
+    source = {
+        'id': 'hra',
+        'name': 'Health Research Authority'
+    }
+    return source
+
+
+def extract_trial(record):
+
+    # Get identifiers
+    identifiers = base.helpers.clean_dict({
+        'hra_internal_identifier': record['application_id'],
+        'iras_project_identifier': record['iras_proj_id'],
+        'study_type_id': record['study_type_id']
```

Comment:
I've checked this field it can take a value as: `00781976`, `NCT00672477`, `n.a`, `N/A`, `0`, `not_known`, should I check before insertion if the field is in the right format `NC`?